cellprofiler/pipeline.py

```diff
@@ -948,7 +949,7 @@ def rl():
                         "You can ignore this warning if you do not plan to save\n"
                         "this pipeline or if you will only use it with this or\n"
                         "later versions of CellProfiler."
-                    ).format(git_hash, pipeline_date)
+                    ).format(pkg_resources.get_distribution("CellProfiler").version)
```

Comment:
This will display the current version of CellProfiler, not the version the pipeline was saved with. You want to display which version of CellProfiler this pipeline was saved with.